docs/articles/apex/testing.md

```diff
@@ -2,27 +2,39 @@
 title: Apex Tests
 ---
 
-You can run Apex tests from within a file or from the Apex Tests sidebar. The sidebar provides other useful features for working with your tests.
+Apex extension allows you run Apex tests to verify the functionality of your code. You can retrieve code coverage results for your Apex classes and triggers every time you run one or more tests. To do this, edit your workspace settings and set `salesforcedx-vscode-core.retrieve-test-code-coverage` to **true**.
```

Comment:
> Should we link the doc on setting workspace settings?

Yeah, I think that's a good idea to link to it @brpowell .  Especially for new users, this might not be familiar and will save them searching